src/main/java/com/github/bartimaeusnek/cropspp/crops/cpp/CobaltBerryCrop.java

```diff
@@ -40,16 +40,8 @@ public ItemStack getGain(ICropTile crop) {
 
     @Override
     public int growthDuration(ICropTile crop) {
-        int r;
-        if (ConfigValues.debug)
-            r = 1;
-        else if (crop.getSize() == 2)
-            r = 3000;
-        else if ((crop.getSize() == 3 && crop.isBlockBelow("blockCobalt")))
-            r = 3000;
-        else
-            r = 500;
-        return r;
+        if (ConfigValues.debug) return 1;
+        return crop.getSize() >= 2 ? 3000 : 500;
```

Comment:
Missing check for size == 3. Intended?

Reply:
Yes, as the code is currently written it always hits on >= 2, so it never hits size == 3 anyway.

Reply:
Oh, for this one, it returns 3000 on both 2 and 3 and the block check is handled by canGrow(), so I just combined them.